assets/src/components/detours/diversionPage.tsx

```diff
@@ -49,6 +49,10 @@ const displayFieldsFromRouteAndPattern = (
   return { routeName, routeDirection, routeOrigin, routeDescription, shape }
 }
 
+const parseIntoDirectionsList = (directions: string) => {
+  return directions.split("\n").map((direction) => ({ instruction: direction }))
+}
+
```

Comment:
I think we should approach this differently and do this in the state machine, hence my hesitancy to approve https://github.com/mbta/skate/pull/2880

I think we should be doing this "parsing" when the `detour.share.edit-directions` event is fired, and assigning that value directly to `directions`, instead of keeping an intermediate `editedDirections` value.